modules/pokemon.py

```diff
@@ -728,34 +728,32 @@ class StatusCondition(Enum):
     Freeze = "frozen"
     Paralysis = "paralysed"
     BadPoison = "badly poisoned"
-
-    def __init__(self, value: str, turns_remaining: int = -1):
-        super().__init__(value)
-        self.turns_remaining = turns_remaining
```

Comment:
Enumerator members are supposed to be constants. The preferred way to define a non-constant would be using `enum.auto`.